sirepo/feature_config.py

```diff
@@ -85,6 +85,15 @@ def for_sim_type(sim_type):
     )
 
 
+def global_sim_cfg():
```

Comment:
Nits: 
- I don't think `cfg` is needed in the name. Everything from this module is cfg.
- What do you think about renaming to be `gui_global` or something similar? All of the cfg values are "global" on the backend so sim confused me. Another option would be to just say YAGNI and put `hide_guest_warning` at the top-level. Then the next person who has a better idea of what the abstraction should be can solve it.

Reply:
> @mkeilman I'm signing off for vacation shortly. I've approved the PR. Do what you will with my comments. Merge when you're ready.

This got past me somehow - I have kept the flag in its own dict but am using ```sim_common``` - we use "common" in the schema e.g. so there is precedent.